xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/xlist.less

```diff
@@ -54,7 +54,7 @@ ul.xlist li.create a {
 }
 
 ul.xlist li.xitem div.xitemcontainer {
-  text-indent: 1.5em;
+  text-indent: 0;
```

Comment:
The ``xlist`` class is probably used in other places. Have you checked all of them to see if this change doesn't break anything? See https://github.com/xwiki/xwiki-platform/search?q=xlist&unscoped_q=xlist .

Reply:
Yes, there is an only single occurrence of  `ul.xlist li.xitem div.xitemcontainer` class in xlist.less with property ` text-indent `.

Reply:
@mflorea Thanks for reviewing. Let me know If there is still any mistake.

Reply:
> There is an only single occurrence of ``ul.xlist li.xitem div.xitemcontainer`` selector across the XWIKI that's in xlist.less with property ``text-indent``.

I'm not sure you understood. The idea is to check the places where the ``xitemcontainer`` class is used on a ``div`` inside a ``li.xitem`` inside a ``ul.xlist``. Basically to check the places that are affected by this change.

Reply:
Oh! Got it. Let me have a look.

Reply:
@mflorea I checked all the places but I didn't find any other place affected by this change. The " Available package" is the only place affected by this change.

Reply:
OK, I trust you. Now, the question is: do you need to set ``text-indent`` to ``0`` or could you simply remove the rule. It makes sense to set to ``0`` only if it inherits a different value.

Take a look at the next rule (below). It looks to me that it overwrites the inherited ``text-indent``, so if you remove the ``text-indent`` from the parent (above) then this rule is not needed anymore either. Would be good to do a quick check.

Reply:
Yes, you are right! I removed the rule `text-indent: 0` because `text-indent` was not inherited from anywhere. I also removed the next rule (below) that was also not inheriting any `text-indent`. So, basically I removed both of them because `text-indent: 0` was not needed anymore.
@mflorea Please have a look at the changes. Thanks